tests/Makefile

```diff
@@ -262,8 +262,13 @@ test-lz4-basic: lz4 datagen unlz4 lz4cat
 	cat tmp-tlb-hw >> tmp-tlb-hw.lz4
 	$(LZ4) -f tmp-tlb-hw.lz4               # uncompress valid frame followed by invalid data
 	$(LZ4) -BX tmp-tlb-hw -c -q | $(LZ4) -tv  # test block checksum
+	test "$(shell ./datagen -g20KB | $(LZ4) -c --fast | wc -c)" -lt "$(shell ./datagen -g20KB | $(LZ4) -c --fast=9 | wc -c)" # compressed size of compression level -1 should be lower than -9
```

Comment:
Lets try this:

```
./datagen -g20KB > tmp-tlb-fast
test "$(shell $(LZ4) -c tmp-tlb-fast --fast | wc -c)" -lt "$(shell $(LZ4) -c tmp-tlb-fast --fast=9 | wc -c)"
```

That way it is clear all the calls are getting the same input, and it is a bit shorter. The `shell` command keeps the current working directory, so it should work (otherwise `$(LZ4)` wouldn't work).